Vagrantfile

```diff
@@ -6,6 +6,196 @@ $NODEMEM=256
 # Overwrite host locale in ssh session
```

Comment:
I'd move chanchable options to a separate config like in https://github.com/openstack-dev/devstack-vagrant/blob/master/Vagrantfile#L7-L12
It's very useful to not have to edit local copy of vagrantfile and/or track down all the places where the value is hardcoded

Reply:
Something to consider though we are trying to keep this experience as straight forward and simple as possible. We purposefully do not want to add too many knobs and switches to avoid issues getting to the task at hand learning how to automate with Ansible. By `curl`'ing a single Vagrantfile followed by a `vagrant up` then `vagrant ssh` it can't get much easier than that.

Ideally the whole bootstrap process would be done by Ansible and not the bash script and embedded files process that @bertvv and @dagwieers are introducing here, but in streamlining this process it makes the most sense.

Hope that reasoning makes sense.

Reply:
Sounds good, but it won't make it more complicated for user if there would be sane defaults and it will work without config, but support changing things in the independent file seamlessly.
You're talking about public switches, but if they're not explicitly advertised to users it shouldn't be harmful and will enable ones who needs to really do changes make them in a comfortable way.